projects/components/src/tabs/navigable/navigable-tab-group.component.ts

```diff
@@ -49,6 +49,10 @@ import { NavigableTabComponent } from './navigable-tab.component';
       </nav>
       <div class="divider"></div>
     </div>
+
+    <!-- When tabs input changes, change detection will be triggered automatically -->
```

Comment:
The way we handle this elsewhere is just wrapping querylists with `queryListAndChanges` util. This should be used everywhere we use contentChildren

Reply:
Fixed.